src/main/resources/openapi/transcriptions.yaml

```diff
@@ -681,7 +764,11 @@ components:
         description:
           type: string
           example: Standard
-
+    UpdateTranscriptions400ErrorCode:
+      type: string
+      enum:
+        - "UT-PP-400"
+      x-enum-varnames: [ UPDATE_TRANSCRIPTIONS_PARTIAL_PROBLEM ]
```

Comment:
Should this be `FAILED_TO_UPDATE_TRANSCRIPTIONS` as seen in TranscriptionApiError.java ?
Maybe I've got the wrong end of the stick with the x-enum-varnames stuff - but thought it was supposed to match that enum TranscriptionApiError

Reply:
no, this enum is referenced in TranscriptionApiError